pom.xml

```diff
@@ -2134,6 +2134,7 @@ under the License.
 								<exclude>org.apache.flink.streaming.api.environment.LegacyLocalStreamEnvironment</exclude>
 								<exclude>org.apache.flink.streaming.api.functions.sink.RichSinkFunction#invoke(java.lang.Object)</exclude>
 								<exclude>org.apache.flink.streaming.api.functions.sink.SinkFunction</exclude>
+								<exclude>org.apache.flink.api.common.io.OutputFormat</exclude>
```

Comment:
ah, so that's why japicmp complained.

Reply:
yes as I said in the commit message as it was not user facing I proposed this exception, but it is just to proposition. If it is not the correct move I can definitely do otherwise. I just did not know what was acceptable: I mean SinkFunctions for example were already in the exceptions.